packages/jest-fake-timers/src/__tests__/jestFakeTimers.test.ts

```diff
@@ -625,6 +625,34 @@ describe('FakeTimers', () => {
       timers.advanceTimersByTime(100);
     });
 
+    it('setTimeout inside setTimeout', () => {
```

Comment:
Should be inside the new `describe`, no?

Reply:
why? it's already in the `advanceTimersByTime` `describe`. I see this is the convention in this file

Reply:
But you're running `advanceTimersToNextTimer`, not `advanceTimersByTime`